BTMLColorLOSMod/Settings.cs

```diff
@@ -10,8 +10,43 @@ public float[] indirectLineOfFireArcColor
             set => IndirectLineOfFireArcColor = new Color(value[0], value[1], value[2], value[3]);
         }
 
+        public float[] blockedLOFLine1Color
```

Comment:
I'd like to see the setters grouped with their backing variable, and comments to explain what ideas all these represent.

Reply:
Updated this but maybe we should wrap the whole section of each "type" in a region? Or nest them in the mod.json? Up to you.